langchain-core/src/utils/testing/index.ts

```diff
@@ -369,9 +374,15 @@ export class FakeListChatModel extends BaseChatModel {
 
   async _generate(
     _messages: BaseMessage[],
-    options?: this["ParsedCallOptions"]
+    options?: this["ParsedCallOptions"],
+    runManager?: CallbackManagerForLLMRun
   ): Promise<ChatResult> {
     await this._sleepIfRequested();
+    if (this.emitCustomEvent) {
```

Comment:
In python this is doable using the regular dispatch API since a run tree will already exist. Why is this not the case here?

Reply:
OK i was wrong this is not dooable on the python side either